app/views/containers/_form.html.slim

```diff
@@ -16,26 +16,26 @@
     .form-group.row
       = source_form.label :source_type, class: 'col-sm-2 col-form-label'
       .col-sm-10
-        = source_form.collection_select :source_type, Source.source_types, :first, :second, {}, id: :container_source_source_type, class: 'form-control'
+        = source_form.collection_select :source_type, Source.source_types, :second, :second, {}, id: :container_source_source_type, class: 'form-control'
     .form-group.row
       = source_form.label :source_mode, class: 'col-sm-2 col-form-label'
       .col-sm-10
         = source_form.collection_select :mode, Source.modes, :first, :second, {}, id: :container_source_mode, class: 'form-control'
-    .form-group.row
-      = source_form.label :source_remote, class: 'col-sm-2 col-form-label'
-      .col-sm-10
-        = source_form.collection_select :remote_id, Remote.all, :id, :name, {include_blank: true}, id: :container_source_remote_id, class: 'form-control'
+    = source_form.fields_for :remote do |remote_form|
+      .form-group.row
+        = remote_form.label :source_remote, class: 'col-sm-2 col-form-label'
+        .col-sm-10
+          = remote_form.collection_select :name, Remote.all, :name, :name, {include_blank: true}, id: :container_source_remote_name, class: 'form-control'
```

Comment:
what if we just send id here instead of name

Reply:
It can but we need to refactor `self.create_with_source` in container model, because this function also called by ext_app API container controller send name.

Reply:
Done. https://github.com/pathfinder-cm/pathfinder-mono/pull/26/commits/0abffb46bf1dc938feecbc976fa4101c151610d5